packages/node_modules/@webex/plugin-messages/src/messages.js

```diff
@@ -351,10 +356,16 @@ const Messages = WebexPlugin.extend({
         activity.actor.emailAddress || activity.actor.entryEmail;
 
       if (event !== SDK_EVENT.EXTERNAL.EVENT_TYPE.DELETED) {
+        const {content, displayName} = activity.object;
         const files = getHydraFiles(activity);
 
         sdkEvent.data.id = constructHydraId(hydraTypes.MESSAGE, activity.id);
-        sdkEvent.data.text = activity.object.displayName;
+        if (content || displayName) {
+          sdkEvent.data.text = content || displayName;
```

Comment:
If this check is needed, you could simplify with: 

```
const text = content || displayname;
if (text) {
...
```

Reply:
I have addressed this change.